client/blocks/image-selector/README.md

```diff
@@ -0,0 +1,61 @@
+Image Selector
+===
+
+Image Selector is a component that uses the Media Modal to select images and preview previously selected images.  It allows multiple images to be passed or it can be limited to a single item. 
+
+## Usage
+
+Pass an array of image IDs that will be shown in the preview and preselected in the Media Modal.
+
+Changes to the selection of images will be passed back in various props for setting images, dropping images, and removing images.  See the example.jsx file for a more complete example.
+
+```jsx
+import ImageSelector from 'blocks/image-selector';
+
+export default function MyFeaturedImages() {
+	return (
+		<ImageSelector imageIds={ [ 34, 105 ] } multiple />
+	);
+}
+```
+
+This component needs both its own styles imported as well as the Media Modal styles on the page in which it's displayed.
+
+```scss
+@import 'assets/stylesheets/sections/media';
+@import 'blocks/image-selector/style';
+```
+
+### Props
+
+Props are displayed as a table with Name, Type, Default, and Description as headings.
+
+Name | Type | Default | Description
+--- | --- | --- | ---
+`className` | `string` | `` | String used for the class of outer ImageSelector wrapper
+`compact` | `bool` | `0` | Determines whether or not the upload picker is compact or full size
+`hasDropZone` | `bool` | `0` | Enables dropzone for drag-and-drop images
```

Comment:
In what cases one would like to disable the drop zone?

Reply:
I think in a compact area (e.g., product variations), the drop zone may not be ideal UX since the drop zone space is pretty limited.  What do you think?

Reply:
Other than not having the drop zone itself, I don't believe it affects the component in any way.  Items are still selectable and uploadable via the MediaModal.

Reply:
I don't think I was clear – what if we eliminated the `hasDropZone` prop and made it depend on the `compact` prop?

```js
const hasDropZone = ! this.props.compact;
```

Reply:
Ah, I see.  That's a good idea.  I'll get this added.

Reply:
Actually on second thought, the compact prop is used to determine the upload picker size, so doing this would disable the drop zone after the first image is selected in the example.

We set the value of compact to true when more than one image id is passed to create a compact uploader for non-featured images.